Application/Source/Model/Trip.swift

```diff
@@ -45,9 +45,8 @@ extension Trip {
             .joined(separator: " - ")
         let countdown: String
         if let begin = begin, Date().isBefore(date: begin, granularity: .day) {
-            let naturalString = try? begin.colloquial(to: Date())
-
-            countdown = " (\(L10n.Trip.List.countdown((naturalString?.colloquial)!)))"
+            let naturalString = begin.colloquial(to: Date())
+            countdown = " (\(L10n.Trip.List.countdown((naturalString ?? "")!)))"
```

Comment:
Why not add the `naturalString` to the `if let` clause as well? This would spare you the nil coalescing as well as the force unwrapping. It also makes more sense to leave the countdown empty rather than the " (begins )" string.